### PR TITLE
feat: [CDS-80035]: Added test coverage for CvStepParametersUtils.java

### PR DESCRIPTION
**Pull request by Anshika2203**
## Describe your changes

Added coverage for CvStepParametersUtils.java

## Checklist
- [x] I've documented the changes in the PR description.
- [x] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/53630)

### CODE REVIEW

File: srm-service/modules/cv-nextgen-service/service/src/test/java/io/harness/cvng/cdng/services/impl/CVNGStepTaskServiceImplTest.java [hunk 1]
```diff
@@ -181,6 +189,31 @@ public void testHealthSources() {
     assertThat(healthSourceDTOSet.iterator().next().getIdentifier()).isEqualTo(cvConfigIdentifier);
   }
 
+  @Test
+  @Owner(developers = ANSHIKA)
+  @Ignore
+  @Category(UnitTests.class)
+  // Working on it.
```
Comment (sribalij) [inline srm-service/modules/cv-nextgen-service/service/src/test/java/io/harness/cvng/cdng/services/impl/CVNGStepTaskServiceImplTest.java:196]: Do you need this statement?

File: srm-service/modules/cv-nextgen-service/service/src/test/java/io/harness/cvng/cdng/services/impl/CVNGStepTaskServiceImplTest.java [hunk 2]
```diff
@@ -40,13 +44,15 @@
 
 import com.google.inject.Inject;
 import java.time.Duration;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Set;
+import java.util.*;
```
Comment (sribalij) [inline srm-service/modules/cv-nextgen-service/service/src/test/java/io/harness/cvng/cdng/services/impl/CVNGStepTaskServiceImplTest.java:47]: Pr review will not let you use *

**Comment by Anshika2203** [comment]
trigger uts

**Comment by Anshika2203** [comment]
trigger messagecheck

**Review by sribalij — DISMISSED** [review]

**Comment by sribalij** [comment]
trigger uts

File: srm-service/modules/cv-nextgen-service/service/src/test/java/io/harness/cvng/cdng/CvStepParametersUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
```
Comment (ankittiwari-harness) [inline srm-service/modules/cv-nextgen-service/service/src/test/java/io/harness/cvng/cdng/CvStepParametersUtilsTest.java:2]: nit: This should be 2023

**Review by ankittiwari-harness — APPROVED** [review]

**Comment by ShubhenduP** [comment]
trigger uts

**Review by sribalij — APPROVED** [review]

File: srm-service/modules/cv-nextgen-service/service/src/test/java/io/harness/cvng/cdng/CvStepParametersUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  @Before
+  public void setup() throws IllegalAccessException {
+    MockitoAnnotations.initMocks(this);
+  }
+
+  @Test
+  @Owner(developers = ANSHIKA)
+  @Category(UnitTests.class)
+  public void testGetStepParameters() {
+    when(stepNode.getName()).thenReturn(STEP_PARAMETER_NAME);
```
Comment (AbhijithHarness) [inline srm-service/modules/cv-nextgen-service/service/src/test/java/io/harness/cvng/cdng/CvStepParametersUtilsTest.java:62]: nit: Step Node is a DTO, you dont need to mock them. Use builder/Factory.

**Review by AbhijithHarness — APPROVED** [review]

**Comment by Anshika2203** [comment]
trigger review

**Pull request merged**